docs/node-mixin/dashboards/use.libsonnet

```diff
@@ -1,275 +1,460 @@
-            ],
-          },
-        )
-      )
-      .addRow(
-        g.row('Disk IO')
-        .addPanel(
-          g.panel('Disk IO Utilisation') +
-          // Full utilisation would be all disks on each node spending an average of
-          // 1 second per second doing I/O, normalize by metric cardinality for stacked charts.
-          // TODO: Does the partition by device make sense? Using the most utilized device per
-          // instance might make more sense.
-          g.queryPanel(|||
-            instance_device:node_disk_io_time_seconds:rate%(rateInterval)s{%(nodeExporterSelector)s}
-            / scalar(count(instance_device:node_disk_io_time_seconds:rate%(rateInterval)s{%(nodeExporterSelector)s}))
-          ||| % $._config, '{{instance}} {{device}}', legendLink) +
-          g.stack +
-          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
-        )
-        .addPanel(
-          g.panel('Disk IO Saturation') +
-          g.queryPanel(|||
-            instance_device:node_disk_io_time_weighted_seconds:rate%(rateInterval)s{%(nodeExporterSelector)s}
-            / scalar(count(instance_device:node_disk_io_time_weighted_seconds:rate%(rateInterval)s{%(nodeExporterSelector)s}))
-          ||| % $._config, '{{instance}} {{device}}', legendLink) +
-          g.stack +
-          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
-        )
-      )
-      .addRow(
-        g.row('Disk Space')
-        .addPanel(
-          g.panel('Disk Space Utilisation') +
-          g.queryPanel(|||
-            sum without (device) (
-              max without (fstype, mountpoint) (
-                node_filesystem_size_bytes{%(nodeExporterSelector)s, %(fsSelector)s} - node_filesystem_avail_bytes{%(nodeExporterSelector)s, %(fsSelector)s}
-              )
-            ) 
-            / scalar(sum(max without (fstype, mountpoint) (node_filesystem_size_bytes{%(nodeExporterSelector)s, %(fsSelector)s})))
-          ||| % $._config, '{{instance}}', legendLink) +
-          g.stack +
-          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
-        ),
-      ),
+                           dashboard.new(
+                             '%sUSE Method / Node' % $._config.dashboardNamePrefix,
+                             time_from='now-1h',
+                             tags=($._config.dashboardTags),
+                             timezone='utc',
+                             refresh='30s',
+                             graphTooltip='shared_crosshair'
+                           )
+                           .addTemplate(datasourceTemplate)
+                           .addTemplate(clusterTemplate)
+                           .addTemplate(
+                             template.new(
+                               'instance',
+                               '$datasource',
+                               'label_values(node_exporter_build_info{%(nodeExporterSelector)s, %(clusterLabel)s="$cluster"}, instance)' % $._config,
```

Comment:
Not setting `%(nodeExporterSelector)s` in this query would enable switching between node_exporter data from different jobs without causing any problems as `node_exporter_build_info` metric should be unique to a node.